dash-pipeline/bmv2/stages/conntrack_lookup.p4

```diff
@@ -12,15 +204,15 @@ control conntrack_lookup_stage(inout headers_t hdr, inout metadata_t meta) {
         @SaiVal[type="sai_dash_flow_enabled_key_t"] dash_flow_enabled_key_t dash_flow_enabled_key,
         bit<32> flow_ttl_in_milliseconds)
     {
-        meta.conntrack_data.flow_table.max_flow_count = max_flow_count;
-        meta.conntrack_data.flow_table.flow_enabled_key = dash_flow_enabled_key;
-        meta.conntrack_data.flow_table.flow_ttl_in_milliseconds = flow_ttl_in_milliseconds;
+        meta.flow_table.max_flow_count = max_flow_count;
```

Comment:
do you mind to keep these fields in the conntrack_data, which makes the code easy the understand and won't complicates the metadata top-level structure - it is already a mess. : (

Reply:
I'll re-organize them in flow metadata with new PR